terraform/09-video-hearing-core/20-main.tf

```diff
@@ -246,6 +246,28 @@ module vh_endpoint {
   tags = local.common_tags
 }
 
+variable "dns_zone_mapping" {
```

Comment:
Should the variable be in a variable.tf file or should this be a local variable?

Reply:
like so? I plopped them here just to make it easy to see where they were coming from. Stuck 'em into inputs-default now